master/getting-started/kubernetes/installation/hosted/index.md

```diff
@@ -5,7 +5,7 @@ title: Calico Kubernetes Hosted Install
 Calico can be installed on a Kubernetes cluster with a single command.
 
 ```
-kubectl apply -f calico.yaml
+kubectl apply -f https://github.com/projectcalico/calico/blob/master/master/getting-started/kubernetes/installation/hosted/calico.yaml
```

Comment:
@anirban37 this may not be obvious and we do plan to rework this content to be more clear, but the idea here of this command is to just describe the syntax of the command. It is not intended for copy/paste purposes. We provide sample commands later on, which use versioned paths. Please note that pulling the YAML from master is not recommended. However, we do thank you for your PR, apologies for the confusion. cc: @ozdanborne